examples/evaluate/04-TBROM_images.py

```diff
@@ -3,21 +3,26 @@
 3D field ROM example for images generation
 ------------------------------------------
 
-This example shows how PyTwin can be used to load and evaluate a Twin model in order to visualize ROM results in
-the form of images with predefined views. The script takes user inputs to evaluate the ROM and will display the
-corresponding image. A first image is generated using the point cloud based ROM Viewer embedded in the Ansys Digital
-Twin Runtime, and a second image is generated by loading the ROM results and performing the post processing on the
-CFD mesh by using PyFluent. Finally, the ROM results are post processing in a 3D Viewer enabled by PyFluent
+This example shows how PyTwin can be used to load and evaluate a twin model to
+visualize ROM results in the form of images with predefined views. The script
+takes the inputs to evaluate the ROM and displays the corresponding image.
+A first image is generated using the point cloud-based ROM Viewer embedded in
+the twin runtime. The second image is generated by loading the ROM results and
+postprocessing the CFD mesh using PyFluent. Finally, ROM results are postprocessed
+in the 3D Viewer enabled by PyFluent.
```

Comment:
Same here--I didn't know if 3D Viewer was a proper GUI name or whether it should be 3D viewer.

Reply:
@PipKat Maybe we should add a link to the PyFluent visualization doc as well?

Reply:
@lboucin Adding a link to the PyFluent visualization is a great idea. Done!